docs/source/guide/tutorials/base/deploy.rst

```diff
@@ -45,4 +55,125 @@ But it can be runned for any task in the same manner.
 We also can deploy the quantized model, that was optimized with NNCF or POT, passing the path to this model in IR format to ``--load-weights`` parameter.
 
 After that, we can use the resulting ``openvino.zip`` archive in other application. 
-Refer to further tutorial :doc:`demo` to visualize the model inference.
\ No newline at end of file
+
+*************
+Demonstrarion
+*************
+
+Using ported demo, we're able to run model in demonstration mode outside of this repository, using only ported ``.zip`` archive with minimum required packages. 
+The demo allows to apply our model on the custom data or the online footage from a web camera and see how it will work in the real-life scenario.
+
+1. Unzip the ``openvino.zip``
+archive.
+
+.. code-block::
+
+    unzip  outputs/deploy/openvino.zip -d outputs/deploy/
+
+2. To run the demo in exportable code, we can use a brand-new virtual environment, 
+where we need to install a minimalistic set of packages required for inference only.
+
+.. code-block::
+
+    python3 -m venv demo_venv --prompt="demo"
+    source demo_venv/bin/activate
+    python -m pip install -r outputs/deploy/requirements.txt
+
+
+3. The following line will run the demo on your input source, 
+using the model in the ``model`` folder. You can pass as ``input`` a single image, a folder of images, a video file, or a web camera id.
+
+.. code-block::
+
+    (demo) ...$ python3 outputs/deploy/python/demo.py --input docs/utils/images/wgisd_dataset_sample.jpg \
+                                                      --models outputs/deploy/model
+
+You can press ``Q`` to stop inference during the demo running.
+
+The model inference on your custom image will look like this:
+
+.. image:: ../../../../utils/images/wgisd_pr_sample.jpg
+  :width: 600
+  :alt: this image shows the inference results on the WGISD dataset
```

Comment:
I have a concern regarding this. As we are preparing this tutorial for all tasks the reader may not know about WGIST dataset and that it was used for OD tutorial. Add a link for the dataset and point out that it is example for object detection model

Reply:
Great point, thank you